<launch>
	<node name="usb_cam_a" pkg="usb_cam" type="usb_cam_node" output="screen" >
           <param name="camera_name" value="usb_cam_a" />
           <param name="video_device" value="/dev/video0" />
           <param name="pixel_format" value="yuyv" />
           <param name="camera_frame_id" value="cam_a_frame" />
           <param name="io_method" value="mmap"/>
           <param name="camera_info_url" value="file:///home/dev3/source/catkin-build/observer/cam_data/cam_a.yaml"/>
	</node>
   
	<node name="image_proc" pkg="image_proc" type="image_proc" ns="usb_cam_a"/>
	
</launch>
